MobileFilter/sections/specific_web.txt

```diff
@@ -4257,6 +4257,13 @@ popularmechanics.com#?#section > div[class*=" "]:has(> div[class]:first-child +
 veranda.com,wetter.com#?#div[class^="sc-"]:has(> div.ad-container)
 wetter.com#?#.som-wrapper:has(> div[data-slotname])
 wetter.com#?#.clickperformance-wrapper:has(> .ad-flex)
+wetter.com#?#.mbanner-fixed:remove()
+wetter.com#$#.mbanner-is-fixed { top: 0px !important; padding-top: 0px !important; }
+wetter.com#$##main-header-fixable { top: 0px !important; }
+wetter.com#?#div.ad-container:remove()
+wetter.com#?#div:has(> div#div-gpt-ad-wetter_direct_toolbar):remove()
+wetter.com#$#body > header > div { padding-top: 0px !important }
+wetter.com#$#body > header > div > div { top: 0px !important; padding-top: 0px !important }
```

Comment:
```
wetter.com#$#body > header > div { padding-top: 0px !important }
wetter.com#$#body > header > div > div { top: 0px !important; padding-top: 0px !important }
```
also can't find where these rules are required.

Reply:
There's an empty space at the top of the home page, because these elements have absolute positioning or padding to allow space for the (now removed) banner:
![image](https://github.com/AdguardTeam/AdguardFilters/assets/57908117/21a5285f-726c-4326-9ed6-20b864f061a4)